index.html

```diff
@@ -48,5 +48,168 @@ <h2 class="prof">I’m a Software Developer</h2>
         </div>
     </div>
 </section>
+
+
+<!-- Recent Works -->
+
+<section id="works-section">
+
+
+  
+  <h3 class="hr">My Recent Work <hr> </h3>
+  
+
+
+<div class="box-container">
+  <div class="left-section">
+    <img src="./img/main (1).png" alt="Recent Project">
+    <div  class="first-desc">
+      <h3>Multi-Post Stories</h3>
+      <P>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</P>
+      <ul>
+        <li>CSS</li>
+        <li>HTML</li>
+        <li>Ruby</li>
+        <li>React</li>
+      </ul>
+      <button type="button">See Project</button>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] Good work styling your main container ✔️. However, I noticed you have a small overflow in the x axis affecting the user experience of your web. 🤓

This is how it looks 👇🏽

https://github.com/Xghayor/Portfolio/assets/116701485/acb4edfb-0142-45a9-9848-49ace0260190